### DISCUSSION TITLE
(solved, due to error of pushgateway ip config )Grafana dashboard shows no data.

### DISCUSSION THREAD

**Discussion by xiaojuan8** [milvus-io/milvus#15719] [Q&A and General discussion]
Hi, I just deploy prometheus and run grafana on my milvus-server. I just import grafana config file of milvus on my grafana dashboard without changing any info.
Then, I tried to search several entities from milvus-server, actually I send 4 request continuously， and got correct response from milvus-server，but there is no data shown on grafana. Can anyone help me?

![image](https://user-images.githubusercontent.com/74544828/155306895-2bb836ef-27be-4872-ab37-f3b6be708b3e.png)

**Comment by xiaojuan8** [comment]
I click refresh on top right of grafana page, nothing changed.

**Comment by LoveEachDay** [comment]
How you do install the milvus cluster? If you install with `milvus-helm` chart, do you enable `metrics.serviceMonitor` ？

**Reply by xiaojuan8** [reply]
Hi, I installed milvus-standlone version. This problem has been solved by changing ip of "metric:ip : server real ip". I set this ip to 127.0.0.1 as default before I changed it. Now it works fine on grafana, data shows on the dashboard. Thanks!
